### PR TITLE
review raster_cutter

### PR DESCRIPTION
**Pull request by hixi**

### CODE REVIEW

File: raster_cutter.py [hunk 1]
```diff
@@ -430,37 +499,45 @@ def delete_world_file(directory_url):
...
 def get_worldfile_url_from_dir(directory_url):
-    index = directory_url.find(".")
+    index = directory_url.find('.')
     if index != -1:
         worldfile_path = directory_url[:index]
-        worldfile_path += ".wld"
+        worldfile_path += '.wld'
     else:
-        raise Exception("Could not find . in path")
+        raise Exception('Could not find . in path')
```
Comment (hixi) [inline raster_cutter.py:511]: This error is technically correct, but can we be more specific, which the underlying problem is (even from the code I don't know, what the issue is). Does this test for path exists and path isdir?

File: raster_cutter.py [hunk 2]
```diff
@@ -430,37 +499,45 @@ def delete_world_file(directory_url):
...
     return worldfile_path
 
+
 def pre_launch_checks():
```
Comment (hixi) [inline raster_cutter.py:515]: This function doesn't seem to be used anywhere, if not required by magic (pyqt), remov it.

**Review by hixi — COMMENTED** [review]
@underwhelmingToaster

File: raster_cutter.py [hunk 3]
```diff
@@ -411,17 +467,30 @@ def generate_lexocad_files(directoryUrl):
...
+            + str(xMinimum)
+            + ' '
+            + str(yMinimum)
+            + '\n'
+            + '# '
+            + str(width)
+            + ' '
+            + str(height)
+            + '\n'
+            + '# projection: EPSG:2056 - CH1903+ / LV95'
```
Comment (underwhelmingToaster) [inline raster_cutter.py:493]: @hixi Was this auto-format? I think the previous version makes more sense because each line is a line in the generated doc.

**Review by underwhelmingToaster — CHANGES_REQUESTED** [review]

**Comment by underwhelmingToaster** [comment]
Thanks @hixi for the review. Implemented suggestions.

**Pull request closed without merging**